src/Mapping/Event/Adapter/ODM.php

```diff
@@ -86,7 +91,20 @@ public function getObjectManager()
             return $this->dm;
         }
 
-        return $this->__call('getDocumentManager', []);
+        if (null === $this->args) {
+            throw new RuntimeException(sprintf('Event args must be set before calling "%s()"', __METHOD__));
```

Comment:
```suggestion
            throw new RuntimeException(sprintf('Event args must be set before calling "%s()".', __METHOD__));
```
BTW, it seems more like a logic exception than a runtime one to me.